sql/src/main/java/io/crate/execution/jobs/Tasks.java

```diff
@@ -48,17 +48,17 @@
 import java.util.stream.Stream;
 
 @Singleton
-public class JobContextService extends AbstractLifecycleComponent {
+public class Tasks extends AbstractLifecycleComponent {
```

Comment:
hm not sure but I like the `Service` suffix as it indicates that this a lifecycle component/a singleton service..